src/index.js

```diff
@@ -1,28 +1,29 @@
 import { Controller } from 'stimulus'
+import { useTransition } from 'stimulus-use'
 
 export default class extends Controller {
   static targets = ['item']
   static classes = ['hidden']
 
   connect () {
     this.class = this.hasHiddenClass ? this.hiddenClass : 'hidden'
+
+    useTransition(this, {
+      element: this.itemTarget,
+      hiddenClass: this.class,
+      transitioned: !this.itemTarget.classList.contains(this.class)
+    })
   }
 
   toggle () {
-    this.itemTargets.forEach(item => {
-      item.classList.toggle(this.class)
-    })
+    this.toggleTransition()
```

Comment:
This does not work with multiple items, right? 

We need to find a solution for both cases